packages/components/src/internal/util/Date.ts

```diff
@@ -97,6 +97,12 @@ export function getUnFormattedNumber(n): number {
     return n ? numeral(n).value() : n;
 }
 
+// Issue 44398: see DateUtil.java getJsonDateTimeFormatString(), this function is to match the format, which is
+// provided by the LabKey server for the API response, from a JS Date object
+export function getJsonDateTimeFormatString(date: Date): string {
+    return _formatDate(date, 'YYYY-MM-dd HH:mm:ss');
```

Comment:
Looking at the note in `DateUtil.getJsonDateTimeFormatString`, is there any concern about this version not including milliseconds?

Reply:
I don't believe so, unless we actually think that users will be entering date/time values with milliseconds on them.